internal/cmd/beta/kms/version/restore/restore.go

```diff
@@ -0,0 +1,117 @@
+package restore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/params"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/kms/client"
+	kmsUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/kms/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/kms"
+)
+
+const (
+	keyRingIdFlag     = "key-ring"
+	keyIdFlag         = "key"
+	versionNumberFlag = "version"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	KeyRingId     string
+	KeyId         string
+	VersionNumber *int64
+}
+
+func NewCmd(params *params.CmdParams) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "restore",
+		Short: "Restore a Key Versions",
+		Long:  "Restores the specified version of key.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Restore key version "0" for the key "my-key-id" inside the key ring "my-key-ring-id"`,
+				`$ stackit beta kms version restore --key "my-key-id" --key-ring "my-key-ring-id" --version 0`),
+		),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			ctx := context.Background()
+			model, err := parseInput(params.Printer, cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(params.Printer, params.CliVersion)
+			if err != nil {
+				return err
+			}
+
+			keyName, err := kmsUtils.GetKeyName(ctx, apiClient, model.ProjectId, model.Region, model.KeyRingId, model.KeyId)
+			if err != nil {
+				params.Printer.Debug(print.ErrorLevel, "get key name: %v", err)
+				keyName = model.KeyId
+			}
+
+			// This operatio can be undone. Don't ask for confirmation!
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("restore Key Version: %w", err)
+			}
+
+			params.Printer.Info("Restored version %d of Key %q\n", *model.VersionNumber, keyName)
```

Comment:
Same here. No valid json/yaml output given. 

And in addition:

```suggestion
			params.Printer.Outputf("Restored version %d of Key %q\n", *model.VersionNumber, keyName)
```

Be aware, `params.Printer.Info()` goes to stderr, while `params.Printer.Outputf()` goes to stdout. 

https://github.com/stackitcloud/stackit-cli/blob/9dfc1f315fc2921537d93e84e36b36fa01a68bbd/internal/pkg/print/print.go#L65-L73

https://github.com/stackitcloud/stackit-cli/blob/9dfc1f315fc2921537d93e84e36b36fa01a68bbd/internal/pkg/print/print.go#L104-L111

Reply:
I understand that you want a json/yaml outputResult() for every call, but how would it look like in this case? There isn't much more information to a restore. The API just returns a 204 ([See docu](https://docs.api.stackit.cloud/documentation/kms/version/v1#tag/versions/operation/post-version-restore)). Inventing a response (like {"status": "success", "action": "restore", "key": ... }) seems overly complicated in this case. 
Could you please give me one clarification example how you expect the output to be formated? Then I'll go ahead and implement it everywhere else.